### DISCUSSION TITLE
What is the "just-issue of deduplicating targets"?

### DISCUSSION THREAD

**Discussion by EliahKagan** [GitoxideLabs/gitoxide#1673] [Q&A] [answered]
What does the

> (and workaround a just-issue of deduplicating targets)

part of

https://github.com/GitoxideLabs/gitoxide/blob/eedebe78f5f780a952d57735c9e6b1a363f0a629/justfile#L18-L19

refer to? Is this saying that it *does* work around such an issue, or that such an issue should be worked around when using it?

This is relevant to splitting the CI `test` job that currently calls `ci-journey-tests` through `ci-test` (as discussed in https://github.com/GitoxideLabs/gitoxide/pull/1668#issuecomment-2474788049 and https://github.com/GitoxideLabs/gitoxide/pull/1668#issuecomment-2475485053), which currently depends on it: if I know the answer to this, then I can avoid making the comments less clear, and possibly make them clearer, when making that change. But it shouldn't block a PR for that if the answer is not known.

In case it's relevant (but maybe it's only relevant to the non-parenthesized part that I am not asking about here), `ci-journey-tests` is currently primarily called through `ci-test`:

https://github.com/GitoxideLabs/gitoxide/blob/eedebe78f5f780a952d57735c9e6b1a363f0a629/justfile#L15-L16

And the four rules `ci-journey-tests` runs are defined as:

https://github.com/GitoxideLabs/gitoxide/blob/eedebe78f5f780a952d57735c9e6b1a363f0a629/justfile#L200-L222

Is the order, with `journey-tests` (for `max`) last, what that parenthesized text in the comment is referring to? It's not clear to me why the order would matter, but it's different from the order in which the dependencies are expressed in `ci-journey-tests`.

**Comment by Byron** [comment, marked as answer]
Thanks for bringing this up.
I believe this is an outdated comment, and I don't think it applies anymore. So just removing the parenthesised portion should do be sufficient.

**Reply by EliahKagan** [reply]
Thanks--I've removed it (among other changes) in #1675.
